db/migrate/20190320155025_change_industry_chp_supercritical_coal_capacity.rb

```diff
@@ -0,0 +1,62 @@
+class ChangeIndustryChpSupercriticalCoalCapacity < ActiveRecord::Migration[5.1]
+  INPUTS = {
+    industry_chp_supercritical_coal: (17.3611111111111 / 600.0)
```

Comment:
The reason nothing happens in this migration is because the input/slider name is "number_of_industry_chp_ultra_supercritical_coal" not "industry_chp_supercritical_coal" (note the "number_of" prefix, and the addition of "ultra"). There is no input or converter called "industry_chp_supercritical_coal".

The first migration can omit the `number_of_` prefix because it also renames `number_of_` to `capacity_of_` (lines 15/16 and 34/35). That's not the case for this second migration: the full input name needs to be provided.

Once this is done, the migration should run as expected.